gen_cmdline.sh

```diff
@@ -847,6 +849,10 @@ parse_cmdline() {
 			CMD_FIRMWARE=$(parse_optbool "$*")
 			print_info 3 "CMD_FIRMWARE: ${CMD_FIRMWARE}"
 			;;
+		--all-firmware|--no-all-firmware)
```

Comment:
This doesn't match the help above.

Reply:
Do you mead `--no-all-firmware`? I did usage lines similar `--*ramdisk-modules`. Should I add only `--no-all-firmware`, or `--no-all-ramdisk-modules` also?

Reply:
Added both to usage.